cli/main.py

```diff
@@ -58,15 +58,18 @@
 
 class NucypherClickConfig:
 
-    def __init__(self):
+    def __init__(self, operating_mode='federated', simulation_mode=False, use_config=False):
 
         self.config_filepath = DEFAULT_INI_FILEPATH  # TODO
-        click.echo("Using configuration filepath {}".format(self.config_filepath))
+        # click.echo("Using configuration filepath {}".format(self.config_filepath))
+
+        self.operating_mode = operating_mode
+        self.simulation_mode = simulation_mode
 
         # Set operating and run modes
-        operating_modes = parse_running_modes(filepath=self.config_filepath)
-        self.operating_mode = operating_modes.get('mode', 'decentralized')
-        self.simulation_mode = operating_modes.get('simulation', False)
+        if use_config is True:
```

Comment:
Just nitpicking, but this expression seems redundant. Perhaps just `if use_config:` ?

Reply:
The same patterns seems to be repeating below.

Reply:
Deprecated upstream, but I generally adopted this pattern in case the user unintentionally passes a non-boolean type.